content/source/docs/enterprise/admin/monitoring.html.md

```diff
@@ -33,7 +33,7 @@ In addition to health-check monitoring, we recommend monitoring standard server
 
 ## Internal Monitoring
 
-Beginning in version 201811-1, Terraform Enterprise includes internal monitoring of critical application metrics using a statsd collector. The resulting metrics are included with the [diagnostic bundles provided to HashiCorp Support](../support/index.html). If the Terraform Enterprise instance is already running a collector on the same port, Terraform Enterprise may not start up correctly due to the conflict, and logs will indicate:
+Beginning in version 201811-1, Terraform Enterprise includes internal monitoring of critical application metrics using a statsd collector. If the Terraform Enterprise instance is already running a collector on the same port, Terraform Enterprise may not start up correctly due to the conflict, and logs will indicate:
```

Comment:
I'm just editing for flow. Let me know if you don't like this or it materially changes the meaning! 

```suggestion
Beginning in version 201811-1, Terraform Enterprise uses a statsd collector to monitor critical application metrics. Terraform Enterprise produces an error during start up if the instance is already running a collector on the same port:
```

Reply:
Thank you! I think it is best to have @sudomateo and @justinretzolk across this.
I am not certain about the intent of this particular section. The PR was only aimed to clarify that the stats are not included in the support bundle.

Reply:
Probably inaccurate to use the term clarify, it is to remove the ambiguity and misunderstanding.

Reply:
I like the suggestion!

Reply:
@hashitop - now that Matthew has weighed in on this, do you feel comfortable making this change? :-)